### ISSUE TITLE
wslview doesn't work anymore after last wsl update changed path 

### ISSUE THREAD

**Issue by Signum21** [wslutilities/wslu#253]
**Background Information:**
 - Windows Version: Windows 10
 - Windows Build: 19045.2311
 - WSL: Kali 2022.4
 - Version: wslu v3.0.1-1; wslview v10

**Describe the bug**
The last update of WSL (1.0.0.0) changed the path from \\wsl$\ to \\wsl.localhost\ and wslview points to the old path when trying to open a file

**To Reproduce**
Open a file with wslview

**Expected behavior**
The file should open in Windows

**Additional context**
Error shown in terminal
```
Start : Access denied
In riga:1 car:128
+ ... oding(850); Start "\\wsl$\kali-linux\mnt\d\Documents\Images\Sfondo.png
+                 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
    + CategoryInfo          : NotSpecified: (:) [Start-Process], UnauthorizedAccessException
    + FullyQualifiedErrorId : System.UnauthorizedAccessException,Microsoft.PowerShell.Commands.StartProcessCommand
```

**Comment by rust84** [comment]
Same issue. Here is the log (Fedora 37). 

```
Start : This command cannot be run due to the error: The system cannot find the file specified.
At line:1 char:128
+ ... putEncoding = [System.Text.Encoding]::GetEncoding(437); Start "lname"
+                                                             ~~~~~~~~~~~~~
    + CategoryInfo          : InvalidOperation: (:) [Start-Process], InvalidOperationException
    + FullyQualifiedErrorId : InvalidOperationException,Microsoft.PowerShell.Commands.StartProcessCommand
```

**Comment by patrick330602** [comment]
Hi @Signum21 ,

You are using an extremely old version. The last available version for Kali is wslu 3.2.4, which is already utilizing the built-in `wslpath`. I suggest you to upgrade to the latest version first; If that still have problem, check `wslpath` utility. If it returns  `\wsl$\`, it would be Microsoft's problem.

Hi @rust84 ,

Your issue is unrelated to his reports and it is reported before; right now I merged them into one and you can post detail there: https://github.com/wslutilities/wslu/issues/256
